include/Framework/EventHeader.h

```diff
@@ -23,6 +23,37 @@ namespace ldmx {
  * @class EventHeader
  * @brief Provides header information an event such as event number and
  * timestamp
+ *
+ * The evolution of the EventHeader object has been pretty slow since
+ * the `*Parameter* members can be used to hold most additional information.
+ * ROOT's serialization infrastructure does define a class version and so
+ * the we document the versions here.
```

Comment:
```suggestion
 * we document the versions here.
```
fix typo in documentation comment